package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bbc-a11y",
-  "version": "2.2.6",
+  "version": "3.0.0",
```

Comment:
Is this really a breaking change?

Reply:
Personally I think so, but happy to downgrade to a minor if that feels better?